src/OpenTelemetry.Instrumentation.Http/TracerProviderBuilderExtensions.cs

```diff
@@ -25,38 +25,30 @@ namespace OpenTelemetry.Trace
     /// </summary>
     public static class TracerProviderBuilderExtensions
     {
-#if NETFRAMEWORK
         /// <summary>
-        /// Enables HttpClient and HttpWebRequest instrumentation.
+        /// Enables HttpClient instrumentation.
         /// </summary>
         /// <param name="builder"><see cref="TracerProviderBuilder"/> being configured.</param>
-        /// <param name="configureHttpWebRequestInstrumentationOptions">HttpWebRequest configuration options.</param>
+        /// <param name="configureHttpClientInstrumentationOptions">HttpClient configuration options.</param>
         /// <returns>The instance of <see cref="TracerProviderBuilder"/> to chain the calls.</returns>
         public static TracerProviderBuilder AddHttpClientInstrumentation(
```

Comment:
This method is here for binary compatibility, but if that is not needed it can be dropped.